documentation/staging/content/userguide/managing-operators/using-helm.md

```diff
@@ -689,8 +689,14 @@ If set to `true`, the `helm install` will install _only_ the conversion webhook
 
 Defaults to `false`.
 
+##### `operatorOnly`
```

Comment:
Two comments:

- The table of contents does not have this, and it looks like like preserveWebhook is missing too - you might want to check all of them. To generate a sample TOC, run `./getlinks.sh userguide/managing-operators/using-helm.md` from within the staging directory.

- Also, IMO, this section would fit better if moved just after "Creating the operator pod" at the same level (or even under it), so that all operations related to basic install/uninstall are near each-other.

Reply:
I have made the suggested changes in [6a08484](https://github.com/oracle/weblogic-kubernetes-operator/pull/3039/commits/6a0848471b62626db8d4eaa46871fb54eb630604).